Updated changelog notes for lakeFS version 0.68.0 (#3535)
treeverse/lakeFS: CHANGELOG.md

# Changelog

## v0.68.0 - 2022-06-21

> 

Reviewer: ```suggestion

This release introduces a bug fix in the garbage collector.
If you used the garbage collection client over a cloud storage that is not S3 (which you shouldn't have done), you'll have to reconfigure it.
If you used the garbage collection client over S3, it will work the same way.

```

What's new:

> - Show content diff for conflicted files (#3522)

Reviewer 2: ```suggestion
- UI: Show content diff for conflicts (#3522)
```

> - Add a new configuration for setting audit logs level (#3512)

Reviewer 2: ```suggestion
- lakeFS configuration for audit log level (#3512)
```

Bug fixes:

> - Fix: Garbage Collector: Eliminate double slash in URL (#3525)

Reviewer 2: ```suggestion
- Fix: Garbage Collector - Eliminate double slash in URL (#3525)
```

> - Fix: Fix initialize logger for superuser cmd auth service crash fix (#3519)

Reviewer 2: ```suggestion
- Fix: Crash fix on `lakectl superuser` command (#3519)
```

# Changelog

## v0.68.0 - 2022-06-21

> This release introduces a bug fix in the garbage collector.

Reviewer 3: ```suggestion
This release fixes a bug in the garbage collector.
```

> If you used the garbage collection client over a cloud storage that is not S3 (which you shouldn't have done), you'll have to reconfigure it.

Reviewer 3: ```suggestion
If you are using cloud storage that is not S3 and have configured a garbage collection policy for retention, you will need to reconfigure it.  (Note that the garbage collector itself does not yet run on such storage!)
```

> If you used the garbage collection client over S3, it will work the same way.

Reviewer 3: ```suggestion
There are no changes if you are using S3.
```
